test/scenarios-bootc/periodics/el96-src@osconfig-lifecycle-and-misc.sh

```diff
@@ -12,5 +12,10 @@ scenario_remove_vms() {
 }
 
 scenario_run_tests() {
-    run_tests host1 suites/greenboot/
+    # Run lifecycle and clusterid tests only
+    # systemd-resolved cannot be merged because it reboots the host
+    # which breaks the SSH connection needed by subsequent suites
+    run_tests host1 \
+        suites/osconfig/lifecycle.robot \
+        suites/osconfig/clusterid.robot
```

Comment:
Let's add `systemd-resolved.robot` at the end of this scenario and set `export TEST_RANDOMIZATION=none`